### PR TITLE
dbus: disable external authentication

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

**Comment by fxcoudert** [comment]
That part seems very weird:

```
@@ -20221,7 +20226,7 @@
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for broken poll" >&5
 $as_echo_n "checking for broken poll... " >&6; }
 if test "$cross_compiling" = yes; then :
-  broken_poll="no (cross compiling)"
+  broken_poll=yes
 else
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
```

File: Formula/dbus.rb [hunk 1]
```diff
@@ -35,6 +36,12 @@ class Dbus < Formula
     sha256 "a8aa6fe3f2d8f873ad3f683013491f5362d551bf5d4c3b469f1efbc5459a20dc"
   end
 
+  # add macports patch to disable external authentication
+  patch :p0 do
+    url "https://raw.githubusercontent.com/macports/macports-ports/master/devel/dbus/files/patch-configure.diff"
```
Comment (DomT4) [inline Formula/dbus.rb]: Using `master` here seems like asking for trouble.

Comment (tschoonj) [reply Formula/dbus.rb]: Fixed

**Comment by tschoonj** [comment]
@fxcoudert Looks harmless to me. I think it was added to avoid trouble while building the universal variant.

**Comment by ilovezfs** [comment]
was this reported upstream?

File: Formula/dbus.rb [hunk 2]
```diff
@@ -35,6 +36,12 @@ class Dbus < Formula
     sha256 "a8aa6fe3f2d8f873ad3f683013491f5362d551bf5d4c3b469f1efbc5459a20dc"
   end
 
+  # add macports patch to disable external authentication
```
Comment (MikeMcQuaid) [inline Formula/dbus.rb]: Why is this patch needed? Can you elaborate more on this both as a GitHub comment and ideally as a code comment? Thanks!

**Comment by tschoonj** [comment]
I opened an issue upstream on the [freedesktop bugzilla](https://bugs.freedesktop.org/show_bug.cgi?id=106545)

**Pull request closed without merging**